subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/EclipseClasspath.java

```diff
@@ -355,4 +361,26 @@ public FileReferenceFactory getFileReferenceFactory() {
         }
         return referenceFactory;
     }
+
+    /**
+     * Returns {@code true} if the classpath contains test fixture classes that should be visible
+     * through incoming project dependencies.
+     *
+     * @since 6.8
+     */
+    @Incubating
+    public boolean getContainsTestFixtures() {
```

Comment:
:x: This should be a getter with a boolean property and there should be no `set` method. 

Maybe a better name for the property would be `includeTestFixtures`? I think that fits how we name APIs in other places better.

Reply:
I just wanted to keep the file consistent. WDYT what should be the strategy? Are we fine with having some properties and some fields mixed in this class?

Reply:
For me it is clear that everything new has to be a property. It's only mixed because we have not migrated the old fields yet. Which we also have to do.

Reply:
Converted the new field.

Reply:
I don't see any change :)

Reply:
Now?